packages/strapi-helper-plugin/lib/src/components/ExtendComponent/index.js

```diff
@@ -0,0 +1,62 @@
+/*
+*
+* ExtendComponent
+*
+*
+*/
+
+import React from 'react';
+import { get, isEmpty } from 'lodash';
+import PropTypes from 'prop-types';
+
+class ExtendComponent extends React.Component {
+  getInjectedComponent = () => {
+    const injectedComponent = this.context.plugins.reduce((acc, plugin) => {
```

Comment:
You don't need to declare this variable. You can return the result of the reduce method directly.